### ISSUE TITLE
Re-use Winston 'timestamp' option

### ISSUE THREAD

**Issue by weekens** [vectordotdev/timber-node#62]
I currently have the following Winston logger configured:

```javascript
this.log = log || new (winston.Logger)({
  transports: [
    new (winston.transports.Console)({
      colorize: false,
      prettyPrint: true,
      timestamp: function() {
        return new Date();
      }
    })
  ]
});
```

If I add `formatter: timber.formatters.Winston` configuration to these options, timestamps in my logs would vanish. If I put `"timestamp_prefix": true` option to `"timber"` section in my `package.json`, the timestamps would be present in logs, but in the wrong format.

I think, Timber should re-use `"timstamp"` option from Winston logger configuration, if it is present.

**Comment by ariamckinley** [comment]
Thanks for the suggestion! I'm looking into this now.

**Comment by ariamckinley** [comment]
This has been resolved via #64 and will be available in the next release.

**Comment by ariamckinley** [comment]
This feature is now live with the latest release `3.0.2`.
